ui/.babelrc

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
Why we don't need it anymore? Is it because of `react-scripts`?

Reply:
Yes. If we want to customize `.babelrc`, we need to run `npm run eject` first.
https://stackoverflow.com/a/48756970